Supporting vendor assessments (#1094)
OctopusDeploy/docs: docs/security/index.md

Every year Octopus undergoes a security review conducted by a third-party company. The latest report can be downloaded: [security-review.pdf](https://octopus.com/documents/security-review-executive-report.pdf)

> We often hear from customers wanting to know more about our security posture. We've performed a [self assessment against various industry standard controls](/docs/security/caiq/index.md). Feel free to use this in any vendor assessments you'd like to perform.

Reviewer: ```suggestion
We often hear from customers who want to know more about our security posture. We've performed a [self assessment against various industry-standard controls](/docs/security/caiq/index.md). Feel free to use this in any vendor assessments you need to perform.
```

Every year Octopus undergoes a security review conducted by a third-party company. The latest report can be downloaded: [security-review.pdf](https://octopus.com/documents/security-review-executive-report.pdf)

We often hear from customers wanting to know more about our security posture. We've performed a [self assessment against various industry standard controls](/docs/security/caiq/index.md). Feel free to use this in any vendor assessments you'd like to perform.

## Responsibility

> Octopus Deploy take responsibility to provide a secure and stable platform for managing your deployments.

Reviewer 2: This doesn't sound quite right what about something like:

"Octopus Deploy has the responsibility to provide a secure and stable platform for managing your deployments.

Author: Thanks @wizedkyle I'm not planning on changing that particular part of the documentation, but it is still a valid question. I usually defer to our wordsmith @wordlee for such things - what do you think Lee?

Reviewer: Mad ea change based on @wizedkyle's suggestion

> Octopus Deploy take responsibility to provide a secure and stable platform for managing your deployments.

Reviewer: Happy to update it. Slight tweak:
```suggestion
Octopus Deploy has the responsibility of providing a secure and stable platform for managing your deployments.
```

> You take responsibility for how that platform is implemented and exposed to your infrastructure and users.

Reviewer 2: Same as above it doesn't sound quite right what about something like:

"You have the responsibility for how that platform is implemented and exposed to your infrastructure and users."

> You take responsibility for how that platform is implemented and exposed to your infrastructure and users.

Reviewer: ```suggestion
You have the responsibility for how that platform is implemented and exposed to your infrastructure and users.
```
